src/visualization/src/MeshcatVisualizer.cpp

```diff
@@ -265,6 +260,18 @@ bool MeshcatVisualizer::setModelState(const iDynTree::MatrixView<const double> &
                                modelName);
 }
 
+bool MeshcatVisualizer::setSphere(const std::string &name,
+                                  const iDynTree::Position &position,
+                                  const double radius,
+                                  const iDynTree::VectorFixSize<3> &color)
+{
+    MeshcatCpp::Material m;
+    m.set_color(color[0] * 255, color[1] * 255, color[2] * 255);
```

Comment:
Can you please document the meaning of the color argument? It is far from being obvious, differently from other arguments of method.

Reply:
Sure @traversaro, I hope it is clearer in this commit a7a7f1db9867502267d75923d7646b54c0bd4f74.